### PR TITLE
Nukeops can be non-human

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Allows nukeops to be non-human

## How This Contributes To The Skyrat Roleplay Experience
Asimov isn't the default lawset here, also with how we handle prefs, most mobs look awful when forced to be human

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
qol: Nukeops can now be any species instead of just human
/:cl:

### CODE REVIEW

**Review by tf-4 — CHANGES_REQUESTED** [review]
It should have something in place for plasmamen, as unlikely as it is to ever be an issue

**Comment by Zonespace27** [comment]
> It should have something in place for plasmamen, as unlikely as it is to ever be an issue

Done

**Review by tf-4 — APPROVED** [review]
lgtm as long as you tested the plasmaman stuff

**Comment by vinylspiders** [comment]
About the plasmaman thing-- I'm actually working on a less hacky solution for this in #18614 . If you're interested, once it's merged you can use one function to take care of all the races that need special items to spawn with for survival (vox, plasmaman).

**Comment by tf-4** [comment]
We can adjust it later

**Pull request merged**
